src/com/palmergames/bukkit/towny/object/Bank.java

```diff
@@ -1,19 +1,46 @@
 package com.palmergames.bukkit.towny.object;
 
+import com.palmergames.bukkit.towny.TownySettings;
 import com.palmergames.bukkit.towny.exceptions.EconomyException;
 import com.palmergames.bukkit.towny.exceptions.TownyException;
 
 /**
  * Allows objects to manage a self contained bank.
+ *
+ * These methods are more of convenience or utility methods.
  */
 public interface Bank extends EconomyHandler {
 	/**
 	 * Takes money from object bank account and gives it to a resident.
 	 * 
 	 * @param resident The resident to pay to.
 	 * @param amount The amount to pay.
-	 * @throws EconomyException Thrown if there is an economy error.
-	 * @throws TownyException Thrown if their is a hierarchical error.
+	 * @throws EconomyException When there's a payment error.
```

Comment:
the apostrophe has to be replaced with whatever character code is need. Or make "there is".

Reply:
what?